frontend/package-lock.json

```diff
@@ -0,0 +1,16976 @@
+{
```

Comment:
Can we not include this in the repository? Maybe add this to gitignore
(The diff is too large in PRs then)

Reply:
It's recommended to not put package-lock.json in  .gitignore
*Source*
https://www.ictshore.com/javascript/package-lock-json-source-control/

Reply:
Oh I know, but what happens is people have some global packages which get added to this lock.json and not package.json (because obviously current project doesn't need it).
Now other people contributing have a lot of other packages and this keeps on jumping.
Technically, it's something that is _not critical_ for the project to work, but is a mess for reviewing and having extra things in the lock.json which you won't need.

Reply:
Done in commit - https://github.com/Open-Sourced-Olaf/SocioMark/pull/35/commits/99065eab5b7fd4bd7176af41aeefe3bc965fcb1f